src/storage-folder.coffee

```diff
@@ -6,11 +6,16 @@ class StorageFolder
   constructor: (containingPath) ->
     @path = path.join(containingPath, "storage") if containingPath?
 
-  store: (name, object) ->
+  storeSync: (name, object) ->
     return unless @path?
 
     fs.writeFileSync(@pathForKey(name), JSON.stringify(object), 'utf8')
 
+  storeAsync: (name, object) ->
```

Comment:
I think this should be called `store`... the convention in Node is to have async be implied and only mention sync vs async on a sync alternative.